velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -46,11 +41,13 @@ struct AggregateWindowTestParam {
   const std::string overClause;
 };
 
```

Comment:
Apologize for a renaming again... But I think AggregatesWindowTest is better as it matches the name of the C++ class as well. Rename both the class/its dependants and file as well.

Reply:
Thanks for the suggestion, updated.